knimin/templates/pm_metagenomics_library_prep.html

```diff
@@ -1,8 +1,84 @@
 {% extends logged_in_index.html %}
 {% block head %}
 <script type="text/javascript">
+  function validateForm(e) {
+    // Deactivate form submission as we are managing what to do here
+    e.returnValue = false;
+
+    $('#pm-submit-btn').prop('disabled', true);
+
+    var formData = new FormData($('#pm-form')[0]);
+
+    $.ajax({
+      url: '/pm_library_prep/metagenomics/',
+      data: formData,
+      processData: false,
+      contentType: false,
+      type: 'POST',
+      success: function(data){
+        var win = window.open('/pm_library_prep/metagenomics/echo/?plate_id={{plate['shotgun_normalized_plate_id']}}&volume=' + formData.get('volume'), '_blank');
```

Comment:
The variable `win` does not seem to be used anywhere. Is it necessary to declare it explicitly?